VulkanExperiment/include/VulkanExperiment/logging.hpp

```diff
@@ -1,11 +1,67 @@
 #pragma once
 
-namespace vke::platform {
+#include <memory>
+#include <string>
+
+/**
+ * @brief In this namespace, i do the impossible, I take on the problem of
+ * dynamic logging. Wish me luck on this super challenge.
+ *
+ */
+namespace vke::platform::logging {
```

Comment:
now that i got boost in here, i gotta figure out how to not leak its logging to the rest of the app.

Reply:
figured it out, had to remove boost, cause its a beast.